core/arch/arm/proc.c

```diff
@@ -96,13 +96,15 @@ DR_API
 int
 proc_num_simd_saved(void)
 {
+    SELF_UNPROTECT_DATASEC(DATASEC_RARELY_PROT);
```

Comment:
Same problem here: this is on the wrong function.

Reply:
See above.